File: benchmarks/base64-decode/main.py
import time
import base64

i1 = "SGVsbG8sIFdvcmxkIQ=="
i2 = "VGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4="
i3 = "QWFCYkNjRGRFZUZmMTIzNDU2XyFAIyYqXigpJD0rLQ=="

before = time.time_ns()

base64.b64decode(i1)
base64.b64decode(i2)
base64.b64decode(i3)

after = time.time_ns()

print(after - before)

File: benchmarks/base64-decode/main.rs
extern crate base64;

use std::time::{SystemTime};
use base64::prelude::*;

fn main() {
    let i1 = "SGVsbG8sIFdvcmxkIQ==";
    let i2 = "VGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4=";
    let i3 = "QWFCYkNjRGRFZUZmMTIzNDU2XyFAIyYqXigpJD0rLQ==";

    let before = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();

    let _ = BASE64_STANDARD.decode(i1);
    let _ = BASE64_STANDARD.decode(i2);
    let _ = BASE64_STANDARD.decode(i3);

    let after = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();
    println!("{}", after.as_nanos() - before.as_nanos());
}